Updating CAPI Overview - Prereqs Section/Additional Links (#1238)
rancher/rancher-docs: docs/integrations-in-rancher/cluster-api/overview.md

![overview](/img/30000ft_view.png)

## Security

> [SLSA](https://slsa.dev/spec/v1.0/about) is a set of incrementally adoptable guidelines for supply chain security, established by industry consensus. The specification set by SLSA is useful for both software producers and consumers: producers can follow SLSA’s guidelines to make their software supply chain more secure, and consumers can use SLSA to make decisions about whether to trust a software package.

Author: Went with your suggestions and updated the Security section, thanks Marty!

> Rancher Turtles meets [SLSA Level 3](https://slsa.dev/spec/v1.0/levels#build-l3) requirements for appropriate build platform, consistent build process, and provenance distribution. For more information, visit the [Rancher Turtles Security](https://turtles.docs.rancher.com/docs/security/slsa) document.

Reviewer: ```suggestion
Rancher Turtles meets [SLSA Level 3](https://slsa.dev/spec/v1.0/levels#build-l3) requirements for appropriate build platform, consistent build process, and provenance distribution. For more information, visit the [Rancher Turtles Security](https://turtles.docs.rancher.com/security/slsa) document.
```

Author: Updated the links, thanks Billy!

- Disables the `embedded-cluster-api` feature in Rancher.
- Deletes the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks, as they are no longer needed.

> The above webhooks can be removed through the Rancher UI as well and are accessed from the left-hand navigation from your local cluster under the **More Resources** > **Admission** section or found by using the **Resource Search** field and inputting the webhook names. Additionally, the following `kubectl` commands can also be used to remove the necessary webhooks:

Reviewer 2: ```suggestion
These webhooks can be removed through the Rancher UI as well:

1. In the upper left corner, click **☰** > **Cluster Management**.
1. Select your local cluster.
1. In the left-hand navigation menu, select **More Resources** > **Admission**.

They can also be accessed by entering the names of the webhooks into the **Resource Search** field.

The following `kubectl` commands can manually remove the necessary webhooks:
```

The instructions need one more step describing what to click to remove the webhooks after the user accesses Admission.

Author: Added a couple more steps to clarify where the "Delete" option can be selected along with your suggestion. Let me know if it makes sense, thanks!

- Disables the `embedded-cluster-api` feature in Rancher.
- Deletes the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks, as they are no longer needed.

These webhooks can be removed through the Rancher UI as well:

1. In the upper left corner, click **☰** > **Cluster Management**.
1. Select your local cluster.
1. In the left-hand navigation menu, select **More Resources** > **Admission**.

> 1. From the dropdown you will see the Resource pages for `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration`, click through both pages to remove the necessary webhooks.

Reviewer 2: ```suggestion
1. From the dropdown, select the Resource pages for `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration`.
```

> 1. On the respective Resource pages, click the **⋮** that are attached to the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks to select the **Delete** option. Once deleted, you should see the webhooks removed immediately.

Reviewer 2: ```suggestion
1. On the respective Resource pages, click the **⋮** that are attached to the `mutating-webhook-configuration` and `validating-webhook-configuration` webhooks and select the **Delete** option.

 After they are deleted, you should see the webhooks removed immediately.
```

> They can also be accessed by entering the names of the webhooks into the **Resource Search** field.

Reviewer 2: ```suggestion
The webhooks can also be accessed by entering the names of the webhooks into the **Resource Search** field.
```
